lib/galaxy/tools/zip_collection.xml

```diff
@@ -1,13 +1,14 @@
 <tool id="__ZIP_COLLECTION__"
-      name="Zip Collection"
-      version="1.0.0"
+      name="Zip"
+      version="1.0.1"
       tool_type="zip_collection">
+  <description>two collections into one</description>
   <type class="ZipCollectionTool" module="galaxy.tools" />
   <action module="galaxy.tools.actions.model_operations"
           class="ModelOperationToolAction"/>
   <inputs>
-    <param type="data" name="input_forward" label="Input Dataset (Forward)" />
-    <param type="data" name="input_reverse" label="Input Dataset (Reverse)" />
+    <param type="data" name="input_forward" label="Input collection 1" help="Example: forward reads"/>
+    <param type="data" name="input_reverse" label="Input collection 2" help="Example: reverse reads"/>
```

Comment:
This literally takes in datasets - not collections - the type is right there. I think this change in inappropriate. Maybe we could add a new attribute to change the label if a tool is fed a collection. 

Short of that though how about label="Input 1" with help of help="Example: forward read(s)"?

Reply:
It takes either datasets or collections

Reply:
You can map over a collection but that is true of every tool, the literal tool takes in datasets and not collections. I suggest we just drop both words to avoid this argument though?

Reply:
It is now simply `Input 1` and `Input 2`